### DISCUSSION TITLE
Unable to get scoreboard working.

### DISCUSSION THREAD

**Discussion by jeepdrvr71** [riffnshred/nhl-led-scoreboard#449] [General]
In the supervisor it shows a fatal error and logs may have information. Any help would be greatly appreciated.

https://paste.ubuntu.com/p/qgHShhxrkT/

**Comment by jeepdrvr71** [comment]
pi@scoreboard: ~ $ pi@scoreboard: ~ $ sb-upgrade
Updating NHL LED Scoreboard
error: Pulling is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.
There was an error updating. Try again later?

This is what I'm getting. Not sure which files I need to remove

**Comment by falkyre** [comment]
I'd wait until a new image is created as there's enough changes that I have 100% confidence that doing the old sb-upgrade won't work properly.  I hope to have a new image by end of this week.

**Reply by jeepdrvr71** [reply]
Thanks again for all your work.
